lib/targetprocess.rb

```
@@ -1,5 +1,36 @@
 require "targetprocess/version"
+require "targetprocess/user_story"
+require "targetprocess/user"
+require "targetprocess/task"
+require "targetprocess/bug"
+require "targetprocess/feature"
+require "targetprocess/project"
+require "targetprocess/release"
+require "targetprocess/request"
+require "targetprocess/testcase"
+require "targetprocess/impediment"
+require "targetprocess/iteration"
+require "targetprocess/comment"
 
 module Targetprocess
-  # Your code goes here...
+ 
+  class << self
+    attr_accessor :configuration 
+  end
+
+  def self.configure
+    self.configuration ||= Configuration.new
+    yield(configuration)
+  end
+
+  class Configuration
+    attr_accessor :domain, :username, :password
+
+    def initialize 
```

do we really need to set @domain, @username and @password in initializer?
we can raise an error if we don't have any of them